src/Cache/Engine/RedisEngine.php

```diff
@@ -82,6 +82,10 @@ public function init(array $config = [])
             return false;
         }
 
+        if (!empty($config['host'])) {
+            $this->config('server', $config['host']);
```

Comment:
Yes, definitely yes. :) I thought I placed this _after_ the `parent::init($config);`, my bad.